commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java

```diff
@@ -398,10 +399,15 @@ public CommerceOrder deleteCommerceOrder(CommerceOrder commerceOrder)
 
 		CommerceAccount commerceAccount = commerceOrder.getCommerceAccount();
 
-		workflowInstanceLinkLocalService.deleteWorkflowInstanceLinks(
```

Comment:
instead of checking account, is better to use the hasWorkflowInstanceLink . method

Reply:
See changes